### ISSUE TITLE
Admin API - Delete API Backend

### ISSUE THREAD

**Issue by brylie** [NREL/api-umbrella#150]
Create a REST endpoint in the Admin API for deleting [API Backends](http://apiumbrella.io/docs/admin-api/#api-backends).

User Story
---
```
As a developer extending API Umbrella Web
I need to delete API Backends via the Admin REST API
so that I can ensure the API Backends collections are consistent with my application
```

**Comment by brylie** [comment]
Possibly related #134

**Comment by GUI** [comment]
Addressed in #152
